source/agora/consensus/validation/Block.d

```diff
@@ -431,7 +431,7 @@ unittest
 
     // don't accept block height 0 from the network
     block.header.height = 0;
-    assert(block.isNotValid(Height(0), Hash.init, null,
+    assert(!block.isValid(Height(0), Hash.init, null,
         Enrollment.MinValidatorCount, Enrollment.MinValidatorCount, checker, findGenesisEnrollments));
```

Comment:
I think this will now log something even though we expect it to fail the check as it is not valid.